src/components/App.js

```diff
@@ -0,0 +1,16 @@
+import './App.css';
+import ButtonPanel from './ButtonPanel';
+import Display from './Display';
+
+function App() {
+  return (
+    <div className="App">
+      <div className="calc-wrapper">
+        <Display />
+        <ButtonPanel />
+      </div>
+    </div>
```

Comment:
You need to wrap both `Display` and `ButtonPanel` with [React.Fragment](https://reactjs.org/docs/fragments.html#short-syntax)
Reason:
We don't want to create an extra element, that's why you need to use fragment, so we are able to render without having to create an extra node wrapping `Display` and `ButtonPanel`.

Screenshot from Pathwright:
![image](https://user-images.githubusercontent.com/31897434/127185558-c385026b-291b-40c6-9e3f-774939db27b0.png)